libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_NRF51822_MKIT/PinNames.h

```diff
@@ -61,7 +61,7 @@ typedef enum {
     p28 = 28,
     p29 = 29,
     p30 = 30,
-//    p31=31,    
+    p31 = 31,    
```

Comment:
That pin is not available for GPIO. How are you using it ? The platform page on mbed follows the gpio pin schematic: https://developer.mbed.org/platforms/Nordic-nRF51822/ (p31 is not highlighted there). Product specification manual says P31 is Digital I/O..